deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -844,7 +844,7 @@ bool IsolateShim::RunSingleStepOfReverseMoveLoop(v8::Isolate* isolate,
     int64_t snapEventTime = -1;
     int64_t snapEventEndTime = -1;
     int64_t origNETime = *nextEventTime;
-    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);
+    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);  // TODO JsTTDMoveMode is no longer 64 bit wide, fix use of moveMode parameter
```

Comment:
could you also run lint rule for modified `*.cc` files? This will definitely complain on no. of characters on line.